### PR TITLE
Display, create Log drains

### LINKED ISSUE #98: logging pages (screen 210, 211)
from #51 

  - [x] log drains index page
  - [x] log drains create page
  - ~~[ ] can delete log drain from index page~~ (see #101)
  - [x] make styles match up
  - [x] DRY similarities in tests
  - [x] display log drain info in stack/-header.hbs (show first log drain's info)

Add a "logging" tab for a page that shows a list of logs, like so:
image

And a "new log" page that allows a user to create a new log drain, like so:
image

### PR DESCRIPTION
**Pull request by bantic**
fixes #98

@mixonic, for you

Displays a list of log drains on the stacks/:id/logging page, with a button to add a new log drain.

Deleting a log drain is not supported yet (see #101).

The stack header shows info about the first log drain. That may need to change later based on feedback.

Looks like:

image

and

image

### CODE REVIEW

**Comment by bantic** [comment]
Hold off on this — the url for POSTing to create a log drain is different than GETting one. I need to make some updates.

**Comment by bantic** [comment]
Ready to go, again. Added 'handle' and 'drainType' attrs to logDrain to pass server-side validations. Form now looks like:

image

File: app/stack/log-drains/new/route.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    createLog: function(log){
+      let route = this;
+
+      log.set('stack', this.modelFor('stack'));
+
+      log.save().then(function(){
+        route.transitionTo('stack.log-drains.index');
+      }).catch(function(e){
+        if (!(e instanceof DS.InvalidError)) {
+          return e;
```
Comment (mixonic) [inline app/stack/log-drains/new/route.js]: `return e` will squash the failed promise. Is that the intent?

Comment (bantic) [inline app/stack/log-drains/new/route.js]: good catch. fixing this...

**Pull request merged**